### ISSUE TITLE
NMP v2 receiving error "Cannot find module 'eslint-plugin-lodash'"

### ISSUE THREAD

**Issue by rla** [gajus/canonical#19]
Installed using

```
npm install canonical -g
```

Installation log:

```
/usr/local/bin/canonical -> /usr/local/lib/node_modules/canonical/dist/bin/canonical.js
npm WARN unmet dependency /usr/local/lib/node_modules/npm/node_modules/npm-registry-client/node_modules/npm-package-arg requires semver@'4' but will load
npm WARN unmet dependency /usr/local/lib/node_modules/npm/node_modules/semver,
npm WARN unmet dependency which is version 5.0.1
canonical@3.1.9 /usr/local/lib/node_modules/canonical
├── eslint-config-canonical-react@1.2.15 (eslint-plugin-react@4.3.0)
├── eslint-config-canonical-lodash@1.0.17 (eslint-plugin-lodash@1.6.5)
├── eslint-config-canonical-jsdoc@1.2.21 (eslint-plugin-jsdoc@2.3.1)
└── eslint-config-canonical@1.6.15 (eslint-plugin-sorting@0.0.1, eslint-plugin-objects@1.1.1, eslint-plugin-babel@3.2.0, eslint-plugin-mocha@2.1.0, eslint-plugin-flowtype@2.2.6, eslint-plugin-no-use-extend-native@0.3.4, eslint-plugin-imp
```

Tried to use on files:

```
canonical lint lib/**/*.js
```

Received error:

```
Error Failed to load plugin lodash: Cannot find module 'eslint-plugin-lodash'
Referenced from: /usr/local/lib/node_modules/canonical/dist/syntaxes/js/eslintrc.json Error: Failed to load plugin lodash: Cannot find module 'eslint-plugin-lodash'
Referenced from: /usr/local/lib/node_modules/canonical/dist/syntaxes/js/eslintrc.json
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:286:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.module.exports.load (/usr/local/lib/node_modules/canonical/node_modules/eslint/lib/config/plugins.js:116:26)
    at Array.forEach (native)
    at Object.module.exports.loadAll (/usr/local/lib/node_modules/canonical/node_modules/eslint/lib/config/plugins.js:138:21)                                                                                                                
    at load (/usr/local/lib/node_modules/canonical/node_modules/eslint/lib/config/config-file.js:502:21)                                                                                                                                     
    at /usr/local/lib/node_modules/canonical/node_modules/eslint/lib/config/config-file.js:388:36                                                                                                                                            
    at Array.reduceRight (native)   
```

According to the install log, `eslint-plugin-lodash` seems to be installed.

Versions:

```
node -v
v4.0.0

npm -v
2.14.2
```

**Comment by gajus** [comment]
A simple fix is to just "npm install esling-plugin-lodash". I will investigate the cause.

**Comment by rla** [comment]
@gajus the simple fix does not work, says the same module is missing.

I also tried project-local installation, without -g option to npm. This got me a bit further, with the fix above working, but then giving similar error for different module, `eslint-plugin-jsdoc`. With project-local setup I ran it as

```
node_modules/.bin/canonical lint lib/**/*.js
```

I kept installing plugins manually but there seems to be high number of them and I eventually gave up. It might be some behavior change it npm. I will try with npm 3.x too.
